go.mod

```diff
@@ -5,8 +5,10 @@ go 1.21.3
 require (
 	github.com/golang-migrate/migrate/v4 v4.17.1
 	github.com/google/uuid v1.5.0
+	github.com/jmoiron/sqlx v1.4.0
```

Comment:
The struct models are cool, but I don't think it's bringing in a new dependency for that. I would prefer to not have sqlx, unless there is some reason we need it to support postgres.